docs/6.x/overview.md

```diff
@@ -87,9 +89,10 @@ Gravity consists of the following components:
 |-------------|---------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `tele`      | The build tool. `tele` is used for building Cluster Images. The enterprise edition of `tele` can also publish Cluster Images into Gravity Hub.  |
 | `tsh`       | The remote access client to authenticate against a SAML/OAuth identity provider and remotely connect to Kubernetes clusters via SSH or Kubernetes API.   |
-| `gravity` <small>service</small> | The Cluster manager agent which runs on every  node. It reduces the complexities of Kubernetes management.
-| `gravity` <small>CLI tool</small>  | The CLI tool to perform high level Cluster administration tasks like expansion, upgrades, etc.  |
-| Gravity Hub | Gravity Hub is a web portal and repository for publishing Cluster Images. Think of Gravity Hub as a catalog of Kubernetes clusters and Kubernetes applications. Gravity Hub is only available in the Gravity Enterprise edition. |
+| `gravity`(agent) | The Cluster manager agent which runs on every  node. It reduces the complexities of Kubernetes management.
```

Comment:
Not sure agent needs to be mentioned - it is part of the functionality of the gravity binary and it more of an implementation detail. What I'd mention is the planet container.

Reply:
@a-palchikov could you suggest some language? As an aside, I noticed in general we don't talk much about Planet.